libs/sdk/src/data-structures/multisig-wallet/implementation.ts

```diff
@@ -173,9 +173,13 @@ export class MultisigWallet {
     this._gatekeeperConfig = gatekeeperConfig;
   }
 
-  public async updateGatekeeperConfig(newGatekeeperConfig: GatekeeperConfig) {
+  public async updateGatekeeperConfig(
+    newGatekeeperConfig: GatekeeperConfig,
+    isLogin?: boolean
```

Comment:
Please merge the two parameters into an object so that one doesn't need to guess what the boolean flag might stand for.